Contribution guide clarifications (#6152)
jdavis/twofactorauth: CONTRIBUTING.md

  }
}
```

> - Keywords must be one of the values listed in [`categories.json`][categories]

Reviewer: ```suggestion
- Keywords must be selected from the values listed in [`categories.json`][categories].
```

[authy]: https://authy.com/
[duo]: https://duo.com/
[googleauthenticator]: https://github.com/google/google-authenticator
[README]: https://github.com/2factorauth/twofactorauth#installing-dependencies-hammer_and_wrench

> [tests]: https://github.com/2factorauth/twofactorauth/blob/master/.github/workflows/repository.yml#L23

Reviewer: Is is really useful to link to the block by giving the line number? I bet that nobody will care to update this.
